README.md

```diff
@@ -342,6 +343,27 @@ elapsed,method,name,url,final_url,redirected,response_time,status_code,success,u
 1294,GET,"/","http://local.dev/","http://local.dev/",false,4,200,true,false,1
 ```
 
+## Load Test Debug Logging
+
+Goose can optionally log details about requests and responses for debug purposes. First, when writing
```

Comment:
The "first" is off here, because there is no 2nd.

Reply:
Thanks, fixed. This got rewritten a few times during development, somehow that got left in from an earlier version.

(I was trying to make it clear that two things are necessary: 1) call `log_debug()` from your load test, and 2) start the load test with `--debug-log-file=foo` for debugging logging to happen.)